src/sentry/search/events/fields.py

```diff
@@ -2763,6 +2826,43 @@ def _resolve_user_misery_function(self, args: Mapping[str, str], alias: str) ->
             alias,
         )
 
+    def _resolve_percentile(
+        self, args: Mapping[str, str], alias: str, fixed_percentile: float = None
+    ) -> SelectType:
+        return Function(
+            f'quantile({fixed_percentile if fixed_percentile is not None else args["percentile"]})',
+            [args["column"]],
+            alias,
+        )
+
+    def _resolve_measurement(self, dividend: str, divisor: str) -> SelectType:
+        return Function(
+            "if",
+            [
+                Function(
+                    "greater",
+                    [self.column(divisor), 0],
+                ),
+                Function(
+                    "divide",
+                    [
+                        self.column(dividend),
+                        self.column(divisor),
+                    ],
+                ),
+                0,
+            ],
+        )
```

Comment:
A few things:
1. `_resolve_measurements` isn't a good name here because it's resolving 3 field aliases that happen to be prefixed with `measurement.`
2. The default value for the else case should be `None` instead of `0`.

Reply:
1. Renamed to `_resolve_division`, hopefully this is a better description since all it's doing is division.
2. Updated default to `None`!